src/db.js

```diff
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '1234',
```

Comment:
Hardcoding a default password in the source code is a security risk. Consider removing the default password or ensuring that environment variables are properly set in all environments to avoid using a hardcoded password.